common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -573,7 +573,7 @@ def available_translations(self, transcripts, verify_assets=True):
 
         # If we're not verifying the assets, we just trust our field values
         if not verify_assets:
-            translations = list(other_lang)
+            translations = list(other_lang) if other_lang else []
             if not translations or sub:
                 translations += ['en']
             return set(translations)
```

Comment:
Disguising arbitrary ordering would not be a particularly good use case for sets.  The benefits of sets over lists as a generic collection are twofold: 

1. faster membership testing:  `if x in a_set` takes a constant amount of time, regardless of the number of elements in the set, whereas `if x in a_list` takes time proportional to the length of the list.
2. uniqueness constraints: `set([1, 2]).add(1)` gives the same set you started with.

I think the alternate approach is a good one.  Though it brings up the broader question (which is probably out of scope for this ticket): Is it safe to assume that the untranslated subtitles are in english?  We have courses where the native version is in other languages?